dask_jobqueue/lsf.py

```diff
@@ -112,6 +92,29 @@ async def _submit_job(self, script_filename):
             return result
 
 
+def set_ncpus(ncpus, worker_cores, logger):
```

Comment:
Same for LSF, couldn't we code this in the template file?

Reply:
It's possible though the templating logic can easily get messy and it is often easier and cleaner to offload more complicated logic to filters (which is exactly what is done here). Additionally, I don't think you can easily log things from within a Jinja template.

Reply:
Yeah, I was thinking of removing the log, which don't seem really useful to me. 

But actually, in this case, we could have kept the `ncpus` and `mem`  settings in the `__init__` method, don't we?

Reply:
Definitely agree regarding `ncpus` and `mem`. They're so simple they should just be in the `__init__`.

The `format_memory` function should probably remain as a filter because of the complexity in `lsf_format_bytes_ceil` and `lsf_detect_units`